DESCRIPTION

```diff
@@ -65,4 +65,4 @@ Config/testthat/edition: 3
 Encoding: UTF-8
 Language: en-US
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.3
+RoxygenNote: 7.3.0
```

Comment:
roxygen is already up to v7.3.1, if that matters to you

Reply:
it does. Maybe i am out of date locally